boards/genericSTM32F334K6.json

```diff
@@ -0,0 +1,36 @@
+{
+  "build": {
+    "core": "stm32",
+    "cpu": "cortex-m4",
+    "extra_flags": "-DSTM32F3 -DSTM32F334x8",
+    "f_cpu": "72000000L",
+    "mcu": "stm32f334k6t7",
+    "product_line": "STM32F334X8"
+  },
+  "platforms": "ststm32",
```

Comment:
Do we really need this field? The board manifest is already located in the platform, so it should work just fine without it. 
(Same for G431)

Reply:
When I try to remove this field. I have somme errors "Could not load boards list" when I try to create a new project.

Reply:
Ah sorry I believe it's because I tested by adding this file in my `.platformio/boards` directory. If I add this in the `
platform-ststm32`directory the problem should be fixed. So yes this line isn't necessary.